templates/content/node--oe-project--full.html.twig

```diff
@@ -0,0 +1,56 @@
+{#
+/**
+ * @file
+ * Project full display.
+ */
+#}
+{% if not content.oe_subject.isEmpty() %}
+  {% set _badges = [] %}
+  {% for _badge in content.oe_subject|field_value %}
+    {% set _badges = _badges|merge([{
+      label: _badge|raw,
+      rounded_pill: true,
+    }]) %}
+  {% endfor %}
+{% endif %}
+{{ pattern('content_banner', {
+  background: 'gray',
+  title: label,
+  content: content.oe_teaser,
+  image: image,
+  attributes: create_attribute().addClass(['ps-0']),
+  badges: _badges,
+}) }}
+{% set inpage_navigation_fields %}
+  {{ content.group_project_details }}
+  {{ content.group_coordinators }}
+  <h3 class="fw-bold mb-4 pt-3">
+    {{ content.oe_summary|field_label }}
+  </h3>
+  {{ content.oe_summary }}
+  <h3 class="fw-bold mb-4 pt-3">
+    {{ content.oe_cx_objective|field_label }}
+  </h3>
+  {{ content.oe_cx_objective }}
+  <h3 class="fw-bold mb-4 pt-3">
+    {{ content.oe_cx_impacts|field_label }}
+  </h3>
+  {{ content.oe_cx_impacts }}
+  <h3 class="fw-bold mb-4 pt-3">
+    {{ content.group_stakeholders.oe_project_participants|field_label }}
+  </h3>
```

Comment:
I see a label for participants, but not for contributors.

Reply:
Could be achieved with a macro perhaps, to render title + content but only if content is not empty.